thrift/shadower.thrift

```diff
@@ -71,6 +71,7 @@ struct ScanWorkflowActivityParams {
   30: optional binary nextPageToken
   40: optional i32 pageSize
   50: optional double samplingRate
+  60: optional shared.FeatureFlags featureFlags
```

Comment:
I don't think this is related? The struct is just the activity input for a system workflow.

Reply:
See the client diff, I'm adding this flag to any place making rpc calls in case there will be a reason to turn on/off some features. 

More than that, I did it so we won't have to track which RPC calls would support this; by default it might be a good idea to enable this logic for all RPC calls. We can chat about this if you'd like.